src/gc/env/gcenv.os.h

```diff
@@ -391,6 +394,15 @@ class GCToOSInterface
     // Return:
     //  Number of processors on the machine
     static uint32_t GetTotalProcessorCount();
+
+    static bool CanEnableGCNumaAware();
+    static bool GetNumaProcessorNodeEx(PPROCESSOR_NUMBER proc_no, uint16_t *node_no);
+    static void* VirtualAllocExNuma(void *lpAddr, size_t dwSize, uint32_t allocType, uint32_t prot, uint32_t node);
```

Comment:
This should be just `bool VirtualCommitNuma(void *address, size_t size, uint32_t node)` similar to how regular VirtualCommit is done. It will avoid the WIndows specific constant on Unix.

Even better may be to just add  `uint32_t node` argument to the existing VirtualCommit and have it default to invalid node (like `(uint32_t)-1`). It may be better to keep just one VirtualCommit API in case we need to add more arguments to it in future.